config/tags.config

```diff
@@ -33,6 +33,11 @@ debian-jdk-releases-slim-tags: debian-slim {{ JDK_RELEASES_VER }}-debian-slim {{
 debian-jdk-nightly-full-tags: debian-nightly {{ JDK_NIGHTLY_VER }}-debian-nightly {{ ARCH }}-debian-{{ JDK_NIGHTLY_VER }}-nightly
 debian-jdk-nightly-slim-tags: debian-nightly-slim {{ JDK_NIGHTLY_VER }}-debian-nightly-slim {{ ARCH }}-debian-{{ JDK_NIGHTLY_VER }}-nightly-slim
 
+debian-slim-jdk-releases-full-tags: debian-slim {{ JDK_RELEASES_VER }}-debian-slim {{ ARCH }}-debian-slim-{{ JDK_RELEASES_VER }}
+debian-slim-jdk-releases-slim-tags: debian-slim-slim {{ JDK_RELEASES_VER }}-debian-slim-slim {{ ARCH }}-debian-slim-{{ JDK_RELEASES_VER }}-slim
```

Comment:
not sure about `slim-slim` tags...I know one `slim` is image name, second is stripped jdk, but sounds a bit confusing

Reply:
Ya, I am not sure the best solution for this. The name scheme the repo has is to add `-slim` on the builds that run the slimming script. 

I 100% agree that `debian-slim-slim` does not look good. @dinogun you have any suggestions? I rather not cause to much trouble for you guys by adding in `debian-slim` builds but I am not sure how you want to handle naming.

Reply:
That does look awkward ... I can only think of `debianslim` as one word. Other options are `slimdebian`, `slim-debian` or `debslim`. `debianslim` seems to make the most sense.

Reply:
@dinogun I am fine with making the change to anything you want. While I do not love any of these names, I think `debianslim` is the best?

You want me to make the changes and repush?